util/command_parser/command_parser_test.go

```diff
@@ -1,25 +1,24 @@
 package command_parser_test
 
 import (
-	"io/ioutil"
-
-	"code.cloudfoundry.org/cli/command/commandfakes"
 	"code.cloudfoundry.org/cli/command/common"
 	"code.cloudfoundry.org/cli/util/command_parser"
 	"code.cloudfoundry.org/cli/util/configv3"
 	"code.cloudfoundry.org/cli/util/ui"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"io/ioutil"
 )
 
 var _ = Describe("Command 'Parser'", func() {
 	var (
 		pluginUI *ui.UI
+		v3Config *configv3.Config
 	)
 	BeforeEach(func() {
 		var err error
-		fakeConfig := new(commandfakes.FakeConfig)
-		pluginUI, err = ui.NewPluginUI(fakeConfig, ioutil.Discard, ioutil.Discard)
+		v3Config = new(configv3.Config)
```

Comment:
Do we need a real `configv3` instead of simple fake in the command parser test?

Reply:
Given that CommandParsers was pulling a v3Config everytime it was created in every execution of BeforeEach, it could be more efficient to create a v3Config separately, pass it as parameter instead and use it in PluginUi as well